run.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
```

Comment:
Added `run.sh`. I actually have not added this to other ingests.
TBH it feels a bit frail that I copy/pasted this and just changed a couple lines for the working directory, but I think this is the best/only way to ensure that I'm running the pipeline locally using ODK. I've run parts of the other ingests using ODK doing a hacky thing and running from the `mondo-ingest` repository.

Reply:
Great!

Reply:
@matentzn This might be a bit hacky, but, if you don't mind, can this be something we add to the `mondo-utils`? Eventually. Maybe it's kind of weird, we could talk about the implementation, but I would be calling the Python script from the CLI, not Python code.

Reply:
You mean, add an ODK wrapper into python? Its not a bad idea actually.. a bit funny.. Did you ever see this done, for example with something like https://pypi.org/project/docker/??? This would be coooool!

Reply:
made an issue: https://github.com/INCATools/ontology-development-kit/issues/908

Reply:
Missed your response before. Glad you like the idea! For my purposes, simply calling `run.sh` using `subprocess.run()` or whatever would be fine. But some native ODK support would be great.

I think I've seen that docker project once... I should really use this instead of `subprocess` in a number of my repos.